### PR TITLE
(tlg0645)-cts and header revisions

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated CTS files and header info. Also question re #2444

### CODE REVIEW

**Comment by lcerrato** [comment]
@AlisonBabeu 
I didn't see this and just did a commit to #2444

**Comment by lcerrato** [comment]
I used Justin Martyr as the author (based on the textgroup).
We may have push the other change before these to avoid conflicts.

File: data/tlg0645/tlg001/tlg0645.tlg001.opp-grc1.xml [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 <?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
-  <teiHeader>
-    
+  <teiHeader>    
     <fileDesc>
       <titleStmt>
-	      <title>Apologia</title>
+	      <title xml:lang="lat">Apologia</title>
         <author>Justinus Martyr</author>
```
Comment (lcerrato) [inline data/tlg0645/tlg001/tlg0645.tlg001.opp-grc1.xml:8]: If we keep Justinus Martyr it should be tagged Latin. I would just change all to **Justin Martyr**.

File: data/tlg0645/tlg001/tlg0645.tlg001.opp-grc1.xml [hunk 2]
```diff
@@ -19,15 +18,19 @@
       <sourceDesc>
         <biblStruct>
           <monogr>
-            <title>S. Iustini Aplogiae Duae</title>
+            <title xml:lang="lat">S. Iustini Aplogiae Duae</title>
             <author>Justinus Martyr</author>
```
Comment (lcerrato) [inline data/tlg0645/tlg001/tlg0645.tlg001.opp-grc1.xml:22]: see above

File: data/tlg0645/tlg003/__cts__.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0645" xml:lang="grc" urn="urn:cts:greekLit:tlg0645.tlg003">
   <ti:title xml:lang="eng">Dialogue with Trypho</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg0645.tlg003.perseus-grc2" workUrn="urn:cts:greekLit:tlg0645.tlg003">
-    <ti:label xml:lang="fre">Dialogue avec Tryphon</ti:label>
-    <ti:description xml:lang="fre">Justin, Dialogue avec Tryphon, Georges Archambault, Picard, 1909</ti:description>
+    <ti:label xml:lang=";at">Dialogus cum Tryphone</ti:label>
```
Comment (lcerrato) [inline data/tlg0645/tlg003/__cts__.xml:4]: the failing error is here

**Review by lcerrato — CHANGES_REQUESTED** [review]

**Comment by lcerrato** [comment]
@AlisonBabeu 
I can stash my changes for now but we need to decide on Justin Martyr versus Justinus Martyr (which should be marked as Latin). I was using the former in keeping with the text group.

I have merged the two files you found in #2444 and made other changes there. I can push after this to avoid conflict.

File: data/tlg0645/tlg001/tlg0645.tlg001.opp-grc1.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/tlg0645/tlg001/tlg0645.tlg001.opp-grc1.xml:8]: Hey @lcerrato thanks for catching this. I will change the names on all the files to Justin Marytr.

File: data/tlg0645/tlg003/__cts__.xml (hunk 3 above)
Comment (AlisonBabeu) [reply data/tlg0645/tlg003/__cts__.xml:4]: Oops, how did I miss that. Thanks so much for catching.

File: data/tlg0645/tlg001/tlg0645.tlg001.opp-grc1.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/tlg0645/tlg001/tlg0645.tlg001.opp-grc1.xml:8]: Sigh, I will change it to Justin Martyr (how can I misspell that word?)

**Review by lcerrato — APPROVED** [review]

**Pull request merged**